src/tests/btree_helpers/btree_test_helper.hpp

```diff
@@ -346,7 +361,16 @@ struct BtreeTestHelper {
     }
 
     void multi_op_execute(const std::vector< std::pair< std::string, int > >& op_list, bool skip_preload = false) {
-        if (!skip_preload) { preload(SISL_OPTIONS["preload_size"].as< uint32_t >()); }
+        if (!skip_preload) {
+            auto preload_size = SISL_OPTIONS["preload_size"].as< uint32_t >();
+            auto const num_entries = SISL_OPTIONS["num_entries"].as< uint32_t >();
+            if (preload_size > num_entries / 2) {
+                LOGWARN("Preload size={} is more than half of num_entries, setting preload_size to {}", preload_size,
+                        num_entries / 2);
+                preload_size = num_entries / 2;
+            }
+            preload(preload_size);
```

Comment:
If preload size is more than actual entries, test crashes, so fixed that with this.

Reply:
this can happen because the default frequency of insert op is slightly more than that of in remove ops.